images/techdocs-cli/context/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM docker.io/library/python:3.10.7@sha256:e9c35537103a2801a30b15a77d4a56b35532c964489b125ec1ff24f3d5b53409 as base
+
+FROM base as python
+
+RUN mkdir /srv/workspace
+WORKDIR /srv/workspace
+
+FROM python as pip-tools
+
+COPY requirements-pip-tools.txt requirements-pip-tools.txt
+RUN pip install \
+    --no-cache-dir \
+    --require-hashes \
+    --verbose \
+    --use-pep517 \
+    --requirement requirements-pip-tools.txt && \
+    rm -rf requirements-pip-tools.txt
+
+FROM python as techdocs-cli
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y apt-transport-https=2.2.4 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN curl -fsSL https://deb.nodesource.com/setup_18.x | bash
```

Comment:
Running random scripts from the internet as root?

Reply:
Does this make more sense: https://github.com/coopnorge/engineering-docker-images/pull/339/files#diff-3a1f19d020dbe21ef63102cfc439e6935335a713bd2b61f9597f2590a87fbbe3R16-R22

```Dockerfile
ENV KEYRING=/usr/share/keyrings/nodesource.gpg
RUN curl -fsSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | gpg --dearmor | tee "$KEYRING" >/dev/null
RUN gpg --no-default-keyring --keyring "$KEYRING" --list-keys

ENV NODE_VERSION=node_18.x
RUN echo "deb [signed-by=$KEYRING] https://deb.nodesource.com/$NODE_VERSION $(lsb_release -s -c) main" | tee /etc/apt/sources.list.d/nodesource.list
RUN echo "deb-src [signed-by=$KEYRING] https://deb.nodesource.com/$NODE_VERSION $(lsb_release -s -c) main" | tee -a /etc/apt/sources.list.d/nodesource.list
```

Reply:
I'm also investigating how to copy this from the official node image.

Reply:
Now implemented by pulling node and npm from the official docker image. I could be a bit brittle if node decides to changes things.